backend/data/seedPrints.js

```diff
@@ -0,0 +1,199 @@
+// backend/data/seedPrints.js
```

Comment:
I understand that you might want to keep your prints in a json file like this as a backup. But once you have seeded the database, you can remove this file from the project to make it more clean and tidy